com.unity.render-pipelines.universal/ShaderLibrary/ShaderQuality.hlsl

```diff
@@ -0,0 +1,22 @@
+#ifndef UNIVERSAL_SHADER_OPTIONS_INCLUDED
+#define UNIVERSAL_SHADER_OPTIONS_INCLUDED
+
+#if !defined (_SHADER_QUALITY_LOW) && !defined (_SHADER_QUALITY_MEDIUM) && !defined (_SHADER_QUALITY_HIGH)
```

Comment:
What about SHADER_HINT_NICE_QUALITY?
If we are not supporting it we should mention it's removed.

Reply:
Where should we mention it?

Reply:
Do we want to keep SHADER_HINT_NICE_QUALITY for other operations such as calculating the smoothness factor for distance attenuation or should that also be controlled by the shader quality?